### ISSUE TITLE
add an Automatically screen reader

### ISSUE THREAD

**Issue by 90barricade93** [90barricade93/FP_EC_AI_BC_24Q2#11]
it would be nice if we could add an Automatically screen reader (txt to voice)
from the point that the user is logged in, the screen reader should be active and start reading the content on screen within the space of {children}:

export default function RootLayout({
  children,
}: Readonly<{
  children: React.ReactNode;
}>) {
  return (
    <html lang="en" style={{ userSelect: "none" }}>
      <body className={`${inter.className} flex flex-col min-h-screen`}>
        <Provider>
          <Header />
          <div className="flex-grow flex flex-col">
            <main className="flex-grow">
              {children}  <<<<<<<--------  HERE 
            </main>
          </div>
          <Footer />
        </Provider>
      </body>
    </html>
  );
}

So it will not read title, header, footer but just generated text.
